config/galaxy.ini.sample

```diff
@@ -611,6 +611,10 @@ nglims_config_file = tool-data/nglims.yaml
 # launch the proxy if it is actually going to be used (e.g. for IPython).
 #dynamic_proxy_manage=True
 
+# As of 16.04 Galaxy supports multiple proxy types. The original NodeJS
+# implementation, alongside a new Golang single-binary-no-dependencies
+# version. Valid values are (node, golang)
+#dynamic_proxy=node
```

Comment:
If you do keep both around, `Dynamic proxy can use` should be sth. like `The NodeJS dynamic proxy can ,,,`.